docs/rules/no-unknown-at-rules.md

```diff
@@ -0,0 +1,61 @@
+# no-unknown-at-rules
+
+Disallow unknown at-rules.
+
+## Background
+
+CSS contains a number of at-rules, each beginning with a `@`, that perform various operations. Some common at-rules include:
+
+-   `@import`
+-   `@media`
+-   `@font-face`
+-   `@keyframes`
+-   `@supports`
+-   `@namespace`
+-   `@page`
+-   `@charset`
+-   `@document`
+-   `@viewport`
```

Comment:
```suggestion
```

`@viewport` is not in the list we get from `css-tree/definition-syntax-data` and I believe it's dropped from the spec.